### DISCUSSION TITLE
please make a detailed text or video on how to do it

### DISCUSSION THREAD

**Discussion by td1794** [wanghaisheng/tiktoka-studio-uploader#57] [Q&A]
instructions unclear

**Comment by wanghaisheng** [comment]
https://github.com/wanghaisheng/tiktoka-studio-uploader/blob/playwright/how-to-upload-youtube.md
@t1e2j3

**Reply by td1794** [reply]
I have problem in 3rd step in
python setup.py install
Where to do it and how .
It always show error @wanghaisheng

**Reply by wanghaisheng** [reply]
download code an under the root directory run

**Reply by wanghaisheng** [reply]
can you try the test-youtube-windows.sh @t1e2j3

**Reply by td1794** [reply]
how do i use the code after  the 4th step
unzip this into anywhere you like，open terminal:
pip install -r requirements.txt
python setup.py install

i am not able to understand how to run it 
explain in detail
@wanghaisheng @stevezheng  @BenJamesAndo @abhishekmj303

**Reply by wanghaisheng** [reply]
all you need to know is how to run a bash 
 unzip this into anywhere you like，open terminal:
bash test-youtube-windows.sh

**Reply by td1794** [reply]
showing this
@wanghaisheng 
test-youtube-windows.sh: line 1: pip: command not found
test-youtube-windows.sh: line 2: pip: command not found
test-youtube-windows.sh: line 3: python: command not found
test-youtube-windows.sh: line 9: python: command not found

**Reply by wanghaisheng** [reply]
you should install python

**Reply by td1794** [reply]
still showing same thing 
and now it is showing
from tsup.youtube.youtube_upload import YoutubeUpload
ModuleNotFoundError: No module named 'tsup'

**Reply by td1794** [reply]
it still shows comand not found please create a youtube video
@wanghaisheng

**Reply by wanghaisheng** [reply]
you can post your screen recording from start  .
or  consider sponsor to get instant help

**Reply by td1794** [reply]
Currently I am a student/underage,when I will grow the YouTube channel will definitely come here to help.
I promise 😊 @wanghaisheng

**Reply by wanghaisheng** [reply]
I  need more info  step by step  so

**Reply by td1794** [reply]
How about I share you my email and you can get me connected via telegram

**Reply by wanghaisheng** [reply]
just upload it to youtube  i can easily access .other way is not that good

**Reply by td1794** [reply]
I did not understand what you saying

**Reply by td1794** [reply]
You mean send a video link to you

**Reply by wanghaisheng** [reply]
just send link  here  and I can watch the video

**Reply by td1794** [reply]
You mean the errors I am facing in the code right

**Reply by wanghaisheng** [reply]
nope .step by step   what you have done and see

**Reply by td1794** [reply]
Will share within 12 hours stay tuned .
It's not like virus, because I am a nob at coding

**Reply by td1794** [reply]
still error 
![Screenshot (1)](https://github.com/wanghaisheng/tiktoka-studio-uploader/assets/119741830/5b2495b6-8151-4ab5-b408-072b4d2057b6)

**Reply by td1794** [reply]
i have done all setps 
like
pip install -r requirements.txt
python setup.py install
pip install ytb-up
downloded bash,ubuntu,python,vsc,
the only error is bash test-youtube-windows.sh

**Reply by wanghaisheng** [reply]
google  how to install python 
google how to vscode git bash 
run  bash test-youtube-windows.sh

**Reply by td1794** [reply]
I tried everything.
When running individual line,at the last line it shows tsup no found.
And when running bash test-youtube-windowd.sh
It shows pip and python commands not found.
Even when they are installed and in system path.
Git bash  also installed,still not working

**Reply by td1794** [reply]
When trying to fix code with chatgpt it shows to use python3 or pip3 and still does not work

**Reply by wanghaisheng** [reply]
if you dont understand what I say, I am no help at all .
I have told you record a full video but you seemed have no idea.

**Reply by td1794** [reply]
@wanghaisheng hi there was ill for  while 
still trying to make it work 
been doing progress
![Screenshot (2)](https://github.com/wanghaisheng/tiktoka-studio-uploader/assets/119741830/1de10ad4-9eaa-4146-a6d8-9ca1cb5cac88)
any idea how to fix this

**Comment by td1794** [comment]
in this section
Update

add test-youtube-windows.sh to help tiktokers

add test-tiktok-manual-sessionId.sh to help youtubers


you put the text wrong 
it should be 
Update

add test-youtube-windows.sh to help youtubers
add test-tiktok-manual-sessionId.sh to help tiktokers
